src/main/resources/templates/activity_lecturer.html

```diff
@@ -16,11 +16,38 @@ <h2 class="pb-2" th:text="${activity.getName()}">Aktywność</h2>
         <div class="text-end">
             <a type="button" class="btn btn-primary" href="index">Powrót do listy kursów</a>
             <a type="button" class="btn btn-primary" th:href="|course?id=${course.getId()}|">Powrót do kursu</a>
+            <a type="button" class="btn btn-primary" th:href="|new_grade?courseId=${course.getId()}&activityId=${activity.getId()}|">Dodaj ocenę</a>
         </div>
     </header>
+
+    <section class="container py-3">
+    <div class="text-end">
+
+    </div>
+    </section>
+
     <p class="pb-3" style="border-bottom: 3px solid #343a40;">
         Tutaj możesz podejrzeć wszystkie oceny swoich podopiecznych w ramach tej aktywności.
     </p>
+
+
+
+    <table class="table table-hover table-bordered bg-light">
```

Comment:
I think we should also display description of the Grade - right now there's no way of distinguishing grades of the same user. Descriptions are empty, but we can add them in DataLoader.